content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -1006,6 +1006,63 @@ status:
   terminating: 3 # three Pods are terminating and have not yet reached the Failed phase
 ```
 
+### Success policy {#success-policy}
+
+{{< feature-state for_k8s_version="v1.29" state="alpha" >}}
```

Comment:
```suggestion
{{< feature-state for_k8s_version="v1.30" state="alpha" >}}
```

Reply:
You can also use:
`{{< feature-state feature_gate_name="JobSuccessPolicy" >}}`

which is data-driven. For reference I got similar suggestion:  https://github.com/kubernetes/website/pull/45506#discussion_r1520622365

Reply:
Oh, good call :) Thank you!

Reply:
Done.